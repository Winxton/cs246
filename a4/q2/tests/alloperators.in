-10 -5 * 400 10 / - NEG NEG ABS -90 ABS +
