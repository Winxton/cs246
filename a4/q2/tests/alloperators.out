(|--((-10 * -5) - (400 / 10))| + |-90|)
= 100
